build/versioning.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       arguments: >
         -SwaggerDir '$(System.DefaultWorkingDirectory)/swagger'
         -AssemblyDir '$(System.ArtifactsDirectory)/deploy/webArtifacts/Microsoft.Health.Dicom.Web.dll'
-        -Version 'v1-prerelease'
```

Comment:
I think we should actually be adding v1 here and leaving v1-prerelease as well. The script is set up to take a list of versions. We want it to still check v1-prerelease and make sure we dont make breaking changes until v1-prerelease is completely not supported anymore